drivers/at86rf2xx/include/at86rf2xx_params.h

```diff
@@ -62,10 +62,17 @@ extern "C" {
 /**
  * @brief   AT86RF231 configuration
  */
+#if defined(MODULE_AT86RFA1) || defined(MODULE_AT86RFR2)
+static const uint8_t at86rf2xx_params[] =
+{
+    0 /* dummy value */
```

Comment:
Hmm, maybe this byte of ROM could be saved some other way. But that could be something to tackle when refactoring the `at86rf2xx` driver.